src/interfaces/IERC20.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+
+interface IERC20 {
+  /**
```

Comment:
You seem to not using this file. You can remove it if it's not used.

Reply:
is being used by src/test/utils/AaveV2Helpers.sol:9:1